Source/WebCore/platform/network/curl/CurlContext.cpp

```diff
@@ -551,13 +551,13 @@ void CurlHandle::enableAcceptEncoding()
 
 void CurlHandle::enableAllowedProtocols()
 {
-    static const long allowedProtocols = CURLPROTO_FILE |
 #if ENABLE(FTPDIR)
-        CURLPROTO_FTP | CURLPROTO_FTPS |
+    static constexpr const char* allowedProtocols = "file,ftp,ftps,http,https";
```

Comment:
I think we don't need both `constexpr` and `const`.

Reply:
And, I think 'static' is unnecessary here.

Reply:
I think 'auto' is enough.
> auto allowedProtocols = ...;

Reply:
Fixed